app/config/FrontendAppConfig.scala

```diff
@@ -45,22 +45,21 @@ class FrontendAppConfig @Inject()(configuration: Configuration) {
   def arrivalFrontendRejectedUrl(arrivalId: ArrivalId)  = s"$declareArrivalNotificationUrlBase/${arrivalId.index}/arrival-rejection"
   def unloadingRemarksRejectedUrl(arrivalId: ArrivalId) = s"$declareUnloadingRemarksUrlBase/${arrivalId.index}/unloading-rejection"
 
-  lazy val authUrl: String              = configuration.get[Service]("auth").baseUrl
-  lazy val loginUrl: String             = configuration.get[String]("urls.login")
-  lazy val loginContinueUrl: String     = configuration.get[String]("urls.loginContinue")
-  lazy val destinationUrl: String       = configuration.get[Service]("microservice.services.destination").baseUrl
-  lazy val referenceDataUrl: String     = configuration.get[Service]("microservice.services.reference-data").baseUrl
-  lazy val departureFrontendUrl: String = configuration.get[Service]("microservice.services.departure-frontend").baseUrl
-  lazy val routerUrl: String            = configuration.get[Service]("microservice.services.testOnly-router").baseUrl
-  lazy val enrolmentKey: String         = configuration.get[String]("keys.enrolmentKey")
+  lazy val authUrl: String          = configuration.get[Service]("auth").baseUrl
+  lazy val loginUrl: String         = configuration.get[String]("urls.login")
+  lazy val loginContinueUrl: String = configuration.get[String]("urls.loginContinue")
+  lazy val destinationUrl: String   = configuration.get[Service]("microservice.services.destination").baseUrl
+  lazy val referenceDataUrl: String = configuration.get[Service]("microservice.services.reference-data").baseUrl
+  lazy val routerUrl: String        = configuration.get[Service]("microservice.services.testOnly-router").baseUrl
+  lazy val enrolmentKey: String     = configuration.get[String]("keys.enrolmentKey")
 
   lazy val nctsEnquiriesUrl: String = configuration.get[String]("urls.nctsEnquiries")
   lazy val loginHmrcService: String = configuration.get[String]("urls.loginHmrcService")
 
   val arrivalRejectedLinkToggle: Boolean      = configuration.get[Boolean]("microservice.services.features.arrivalRejectedLink")
   val departureJourneyToggle: Boolean         = configuration.getOptional[Boolean]("microservice.services.features.departureJourney").getOrElse(false)
-  private val departureLRNUrl: String         = configuration.get[String]("microservice.services.departure-frontend.lrnUrl")
-  val declareDepartureStartWithLRNUrl: String = s"$departureFrontendUrl/$departureLRNUrl"
+  private val departureFrontendUrl: String    = configuration.get[String]("urls.declareTransitMovementDepartureFrontend")
+  val declareDepartureStartWithLRNUrl: String = s"$departureFrontendUrl/local-reference-number"
```

Comment:
@PSouthern placing local-reference-number URL in the conf file gives the flexibility to change the URL in future without the need of a code change in application